ruby/object_oriented_programming_basics/object_oriented_programming.md

```diff
@@ -26,7 +26,7 @@ This section contains a general overview of topics that you will learn in this l
     - [Ruby Explained: Inheritance and Scope](http://www.eriktrautman.com/posts/ruby-explained-inheritance-and-scope)
 1. Read the article [Object Relationships in Basic Ruby](https://medium.com/@marcellamaki/object-relationships-in-basic-ruby-1af5773fff48) to see an example of how two classes can interact.
 1. Read the [Bastard's Chapter on Error Handling](http://ruby.bastardsbook.com/chapters/exception-handling/) to reinforce your understanding of dealing with errors.
-1. Do [Quiz #5](http://www.codequizzes.com/ruby/beginner/intro-object-oriented-programming) and [Quiz #7](http://www.codequizzes.com/ruby/beginner/modules-classes-inheritance) from Code Quizzes
+1. Do this [quiz on introductory object-oriented programming](http://www.codequizzes.com/ruby/beginner/intro-object-oriented-programming) and this [quiz on modules, classes, and inheritence](http://www.codequizzes.com/ruby/beginner/modules-classes-inheritance) from Code Quizzes.
```

Comment:
Oops, my typo.

```suggestion
1. Do this [quiz on introductory object-oriented programming](http://www.codequizzes.com/ruby/beginner/intro-object-oriented-programming) and this [quiz on modules, classes, and inheritance](http://www.codequizzes.com/ruby/beginner/modules-classes-inheritance) from Code Quizzes.
```

Reply:
Okay. Will do.

Reply:
In addition to the typo fix above, could you also make the following quick change?

### `object_oriented_programming.md`

1. Remove line 52
2. Unindent the next 2 lines (so they're part of the main list and not a nested list)
3. Amend the "99 Bottles of OOP"'s last sentence to say
   ```text
   This is a great resource if, after completing several OOP projects, you still feel uncertain about OOP concepts and would like to be guided through some refactoring examples and explanations.
   ```

Reply:
Okay. No problem.

Reply:
Quick question: after I made the changes, should I run the linter the same as I did for the lesson, obviously on a different path, or not?

Reply:
Run the `lint:lesson` script with the appropriate file path to see if there are still any lint errors. If 0, you're good to push. If any remain, read the errors and fix accordingly. Most errors are fixable with the appropriate `fix:*` script, but some errors won't be fixable by it and need to be manually fixed based on context.